students/TinaB/lesson04/katafourteen.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import random
+
+# book_text_file = "sherlock.txt"
+#"sherlock_small.txt"
+text_file = open("sherlock_small.txt", 'r')
```

Comment:
It's recommended to use `with` statement for file reading and writing, and it will automatically close the file for you when you exit the `with` block